src/guides/Tech_Demo/README.md

```diff
@@ -0,0 +1,49 @@
+
+## Purpose
+
+The work you did will have achieved something, whether it's fixing a bug, solving a problem for your users, or presenting an entirely new idea.
+
+It's important to show this if you want to convince others that your work is important. Feel free to show the existing problem and then go into how you have addressed it, this will make the change that much more impactful.
+
+## Presentation
+
+Live demos are always best, it proves to your audience that it actually works and that there's no smoke and mirrors hiding flaws. However, there are cases where live demos don't work out. Maybe you have a very slow running process, or your code just decides it's not going to work that day.
+
+For slow processes, explain that it's slow and why, and let the audience know how you've addressed it in the demo to set the right expectations.
+
+Here are some methods:
+
+1. Run the process before starting the demo in another tab/instance/device, and switch to that when you need to show what happens after it.
+2. Do something else while it's running, e.g. talk about some other detail in your presentation.
+3. Pre-record the whole demo and speed up or cut out the slow parts.
+
+It's a good idea to have a well-edited pre-recorded demo to switch to as well, sometimes code just decides not to work!
+
+Make sure you've practiced talking over the pre-recorded demo as well as the live demo so you're prepared for all cases, including switching from live to recorded halfway through.
+
+### Avoid Distractions
+
+It's not the end of the world, but a notification popping up on your screen is distracting to the audience.
+
+It's best to fully close all unneeded apps, turn on do-not-disturb, and full-screen what you're presenting.
+
+This also goes for browser tabs and bookmarks, have everything as clean and empty as possible so all the focus is on what you're presenting.
+
+Use Virtual Desktops on Windows and full-screen apps on MacOS. Place your presentation in one, and the demo in another, that way you can quickly and seamlessly switch between the two without alt-tabbing or exiting the presentation.
+
+## Prepare
+
+Most importantly, rehearse the demo! Go through everything you want to show off, and have a good idea of what you want to say. It doesn't need to be perfectly memorized, but you should know exactly what you're doing next all the time. If you're working with a team, and one person is speaking and the other controlling the demo, make sure both of you are in sync. If you say "looking at the user profile page", the demo should immediately go there, without you having to ask your partner to do things which breaks the flow.
+
+Finally, if (and when) something goes wrong, the best thing to do is to move past it. Bringing more attention to an issue can make it seem worse than it is. If things are really bad (app fully crashes, laptop catches fire), you have a recorded demo to switch to. Right?
+
+
+#### References/Further Reading
```

Comment:
This one’s good too https://www.forbes.com/sites/forbestechcouncil/2019/12/12/13-demoing-strategies-that-make-tech-software-compelling/?sh=5509768b69ac